src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/client/transformation/model/response/ResponseCaseDetails.java

```diff
@@ -2,15 +2,15 @@
 
 import com.fasterxml.jackson.annotation.JsonProperty;
 
-public class CaseCreationDetails {
+public class ResponseCaseDetails {
```

Comment:
We will need to create a separate class for representing case update response. It won't have `case_type_id` and `event_id`, because our service knows which ones we will use.

Reply:
If we have a separate class, it could be named sth like CaseUpdateDetails.